redbot/cogs/mod/mutes.py

```diff
@@ -207,6 +207,14 @@ async def voice_mute(self, ctx: commands.Context, user: discord.Member, *, reaso
             await ctx.send(
                 _("Muted {user} in channel {channel.name}").format(user=user, channel=channel)
             )
+            try:
+                await user.move_to(channel)
+            except discord.Forbidden:
```

Comment:
This should be in an `if` checking whether the bot has that permission so that we don't make API request if we already know that it's gonna fail.
```suggestion
                if channel.permissions_for(ctx.me).move_members:
                    await user.move_to(channel)
                else:
                    raise RuntimeError
            except (discord.Forbidden, RuntimeError):
```